MapboxDirections/MBVisualInstructionType.swift

```diff
@@ -19,9 +19,9 @@ public enum VisualInstructionComponentType: Int, CustomStringConvertible {
     case text
     
     /**
-     Component contains an image that should be rendered.
+     Component either contains an image that should be rendered, or text that should be rendered as a generic icon.
      */
-    case image
+    case icon
```

Comment:
I’m not sure this change is strictly necessary. “Image” and “icon” are roughly synonymous in this context. We went with `image` for consistency with the `VisualInstructionComponent.imageURL` property, which is still present regardless of the API change.